nemo/collections/nlp/parts/megatron_lr_schedulers.py

```diff
@@ -0,0 +1,17 @@
+from nemo.core.optim.lr_scheduler import AVAILABLE_SCHEDULERS, CosineAnnealing
```

Comment:
Please add the copyright here, you can get it from any file in nemo and set the year to 2023